content-repo/extra-docs/articles/setup-google-maps.md

```diff
@@ -0,0 +1,47 @@
+---
+id: google-maps
+title: Setup Google Maps in Cortex XSOAR using an Automation
```

Comment:
Change to something like: "Setup Google Maps in Cortex XSOAR to Use Map Automations"

Otherwise, it makes it sound like you should use an automation to perform the setup, which is not the case.